.gitignore

```diff
@@ -4,3 +4,4 @@ docker/sgx
 make.log
 cscope.files
 cscope.out
+build/__tools__/run-tests-db.sh
```

Comment:
why is this filename in .gitignore? It's not a build artifact (and certainly shouldn't be in __tools__ if so?) and who knows somebody at some point wants to create such a file and will be puzzled because it doesn't work?  

Related to this, can you add a scheme definition of the file? It's pretty burried in the code?

Reply:
Will fix this.